moped-etl/arcgis/settings.py

```diff
@@ -5,8 +5,6 @@
 COMPONENTS_QUERY_BY_LAST_UPDATE_DATE = """
 query GetProjectsComponents($where: component_arcgis_online_view_bool_exp!) {
   component_arcgis_online_view(where: $where) {
-    completion_date
-    completion_end_date
```

Comment:
👍 just a reminder to delete these columns from the AGOL layers as well

Reply:
ah yes, thanks for helping me stay on top of these! 🙏 They are added to an issue to do a bulk removal. 🧹